src/i18n/en-US/nda.ts

```diff
@@ -7,6 +7,7 @@ const nda = {
   accepted: 'Accepted on ',
   header:
```

Comment:
Header is not updated to match the new message

![image](https://github.com/CMSgov/mint-app/assets/2349702/0a6e2867-d7d0-4ab9-b66b-7bb832d5dc1f)

Reply:
![image](https://github.com/CMSgov/mint-app/assets/2349702/f9588013-48be-491c-a30e-16038b77d96d)

Reply:
I'm not sure why the clickthrough was failing for you before. I was unable to make it fail on local testing. The structure of logic has changed significantly due to other backend changes. I cannot find a fail case in the current revision. I'll keep an eye out to see if there are any further issues.